docs/reference/cluster/health.asciidoc

```diff
@@ -1,18 +1,44 @@
 [[cluster-health]]
 === Cluster Health
 
-The cluster health API allows to get a very simple status on the health
-of the cluster. For example, on a quiet single node cluster with a single index
-with one shard and one replica, this:
+[[cluster-health-api-request]]
+==== {api-request-title}
+
+`GET _cluster/health` +
```

Comment:
I think this should be something like
` GET _cluster/health/{index}`

I don't think we need a separate example for comma-separated values.